src/lib/php/common-menu.php

```diff
@@ -196,11 +197,11 @@ function menu_insert_r(&$menuItems, $path, $pathRemainder, $LastOrder, $Target,
   if ($menuItemsExist) {
     foreach($menuItems as &$menuItem) {
       // need to escape the [ ] or the string will not match
-      if (!strcmp($menuItem->Name, $pathElement) && strcmp($menuItem->Name, "\\[BREAK\\]")) {
```

Comment:
Did this really work as intended before? Before the change only one [BREAK] element per submenu was possible.